roles/reset/tasks/main.yml

```diff
@@ -50,14 +50,7 @@
   systemd:
     daemon_reload: yes
 
-- name: Remove linux-modules-extra-raspi
-  apt: name=linux-modules-extra-raspi state=absent
-
-- name: Remove tmp director used for manifests
+- name: Remove tmp directory used for manifests
   file:
     path: /tmp/k3s
     state: absent
-
-- name: Reboot and wait for node to come back up
```

Comment:
@timothystewart6 I did not remove it, I just moved it from the reset role to the reset playbook. I think the reason was this:

 - As the `raspberrypi` role does now contain its own cleanup code rather than relying on `reset` for that, the `reset` playbook does now contain _two_ roles.
 - Also, teardown should happen in the reverse order as setup. So you would first need to stop the k3s processes and then you can also tear down the requirements (in this concrete case: uninstall some kernel modules).
 - Still, you would only want to reboot one time. But if the reboot was part of the `reset` _role_ it would happen in the middle of teardown, not at the end.
 - So I moved it to the `reset` _playbook.

That being said, I'm just seeing now that the role order in the playbook is actually incorrect 🙈 see: #104

Reply:
Ah!  Thank you!  Sorry, I overlooked it but it exposed another bug!  Thanks!